src/main/java/org/threadly/db/aurora/NonRegisteringDriver.java

```diff
@@ -0,0 +1,69 @@
+package org.threadly.db.aurora;
+
+import java.sql.Connection;
+import java.sql.DriverPropertyInfo;
+import java.sql.SQLException;
+import java.sql.SQLFeatureNotSupportedException;
+import java.util.Properties;
+import java.util.logging.Logger;
+
+import org.threadly.db.AbstractArcDriver;
+
+/**
+ * Threadly's AuroraArc NonRegisteringDriver. This Driver will create multiple connections for each returned connection
```

Comment:
If we end up keeping this, we should describe how this is different from the normal `Driver`, and of course point people to it.